Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -4437,3 +4462,44 @@ webkit_settings_set_webrtc_udp_ports_range(WebKitSettings* settings, const gchar
     UNUSED_PARAM(udpPortsRange);
 #endif
 }
+
+/**
+ * webkit_settings_get_inspector_maximum_resources_content_size:
+ * @settings: a #WebKitSettings
+ *
+ * Gets the #WebKitSettings:inspector_maximum_resources_content_size property.
```

Comment:
I see an inconsistency between how the property is linked here and below in the setter, I am not sure which one is correct, but maybe use the same in both comments.